FundacionParaguaya/MentorApp
# Restart draft

**Describe the changes**
Draft remembers where you exit and takes you back to this place when restating it. Every screen knows how to nagivate to the following and the previous screen inside the taking a lifemap process.

**To Test**
Specific steps to test when reviewing:
Go to every screen in the lifemap process and exit the draft.
Go to dashboard, click on the draft and click resume draft. It should go to the screen you left off.
From every screen, try going back and forth - should go to the correct screen.
Important note - test this for every single screen!
